docs/customizing.md

```diff
@@ -0,0 +1,238 @@
+      default_file_storage: 'storages.backends.azure_storage.AzureStorage'
+```
+
+`pulp_workers`, is set to `4`, which means that 4 worker processes will be started for processing jobs on a multi-core system. It replaces the default value of 2.
+
+`pulp_install_plugins` does the following:
+* Directs the installer to install all 3 plugins, pulp-deb, pulp-file and pulp-container.
+* If pulp-container is not installed, the latest version will be installed.
+* If pulp-container is already installed, the current version will be left intact.
+* If pulp-deb is not installed, the latest version will be installed.
+* If pulp-deb is already installed, it will be upgraded to the latest version.
+* If pulp-file is not installed, version `9.8.7` will be installed.
+* If pulp-file is already installed, it will be upgraded to `9.8.7`.
+* pulp-file's static files will be collected at install time. (`true` is the default behavior for every plugin, and pulp-file requires this. So it did not need to be specified. However, it is listed for syntax demonstration purposes, and sometimes plugin will instruct you to set it to `false`.)
+
+
+`pulp_settings` does the following:
+* It generates Pulp's runtime configuration file, `/etc/pulp/settings.py`, with all of the following configured behavior.
+  * `default_file_storage` and `azure_*` configure pulp to point to an object storage on Azure for
+    storing content (rather than defaulting to using the local filesystem.)
+  * It sets the `secret_key` for the Django framework (which) Pulp is written in) to `"secret"`. Note that this setting is always required to be set by the user.
+  * It sets `content_origin`, which specifies the Pulp application's URL, to the FQDN (`hostname -a`) of the Pulp server (beginning with `https://`). Note that this setting is always required to be set by the user, although this example value (with variables in it) is suitable for most users.
+
+More information about `pulp_settings`
+* Please do not modify `/etc/pulp/settings.py` manually after install, because re-running the installer for upgrades / repairing Pulp will revert your changes. Please re-run the installer instead with your changes added to `pulp_settings`.
+* `pulp_settings` is used during both the installation process (it can cause errors at install time with incorrect values), and once pulp is installed.
+
+Explanation of plugin version compatibility with Pulpcore
+--------------------------------------------------
+
+Pulp 3 has a plugin architecture so that new content types and features can be added by the
+larger community. However, both pulpcore & plugins are installed by the installer using `pip`, which has limited
+dependency resolution. Plugins release at their own lifecycles. Thus in the worst case scenario, the
+2 stable branches of plugin pulp_juicy could depend on the current branch of pulpcore, while the 2
+stable branches of pulp_sugary could depend on an older branch of pulpcore.
```

Comment:
To me it's not obvious, what "2 stable branches" you are referring to here.
Shouldn't it be the stable version of the plugin that is compatible with (at least) two minor versions of pulpcore?

Reply:
Rephrased.